### ISSUE TITLE
Exported games crash when building maps at runtime

### ISSUE THREAD

**Issue by DeerTears** [QodotPlugin/Qodot#22]
**Godot Version:** Godot 4.0 mono stable win64
**.NET version:** 7.0.21

Building a map at runtime in an exported game crashes the game with the console error:

```
ERROR: .NET: Failed to load hostfxr.
at: (modules/mono/mono_gd/gd_mono.cpp:399)
```

All games with QodotMap nodes seem to print this error, but thankfully no crash occurs so long as `build_map()` is not called. This error does not happen in the editor when building maps.

**To reproduce:**
1. Install .NET 7.0.21 from https://dotnet.microsoft.com/en-us/download
2. Build C# solution for empty Godot project 
3. Download Qodot from this repository
4. Enable Qodot in Project > Project Settings > Plugins
5. Set map file (example `res://addons/qodot/example_scenes/0-visuals/1-textures/1-textures.map`) and textures directory (example `res://addons/qodot/textures/`) in QodotMap node
6. Set qodot_map.gd const `DEBUG` to `true`, causing the loaded map to build at runtime
7. Export game to windows with console
8. Run game, observe error: 
![image](https://user-images.githubusercontent.com/47726614/223243605-ddd4479c-77fd-4782-9c5f-f27611be5b28.png)

This seems to be caused by an issue with Godot and .NET builds crashing at runtime: https://github.com/godotengine/godot/issues/66353

It seems like there's not much we can do to solve this without re-writing parts of the plugin to target .NET 6 instead of .NET 7, but by the time we get it working Godot may have fixed this issue already and start to more broadly adapt .NET 7's features.

**Comment by EMBYRDEV** [comment]
Some progress on this issue. It seems like dotNet publish is failing with the following code when exporting the game.

```
Build FAILED.

       "C:\Users\Embyr\Documents\QDTEST\QDTEST.csproj" (Publish target) (1:7) ->
       (CoreCompile target) -> 
         C:\Users\Embyr\Documents\QDTEST\addons\qodot\src\import_plugins\QuakeWadImportPlugin.cs(9,45): error CS0246: The type or namespace name 'EditorImportPlugin' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\Embyr\Documents\QDTEST\QDTEST.csproj]

    0 Warning(s)
    1 Error(s)
```

As the C# solution relies on the editor tools being packaged it seems.

**Comment by EMBYRDEV** [comment]
Fixed by e90564b
